RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronSeeds_cfi.py

```diff
@@ -0,0 +1,52 @@
+import FWCore.ParameterSet.Config as cms
+
+def thresholds( wp ) :
+    return cms.vdouble([{"L": 1.03,"M":1.75,"T":2.61}.get(wp,1.e6),  # unbiased
+                        {"L":-0.48,"M":0.76,"T":1.83}.get(wp,1.e6)]) # ptbiased
+
+lowPtGsfElectronSeeds = cms.EDProducer(
+    "LowPtGsfElectronSeedProducer",
+    tracks = cms.InputTag("generalTracks"),
+    pfTracks = cms.InputTag("lowPtGsfElePfTracks"),
+    ecalClusters = cms.InputTag("particleFlowClusterECAL"),
+    hcalClusters = cms.InputTag("particleFlowClusterHCAL"),
+    EBRecHits = cms.InputTag('reducedEcalRecHitsEB'), 
+    EERecHits = cms.InputTag('reducedEcalRecHitsEE'),
+    rho = cms.InputTag('fixedGridRhoFastjetAllTmp'),
+    BeamSpot = cms.InputTag("offlineBeamSpot"),
+    Fitter = cms.string('GsfTrajectoryFitter_forPreId'),
+    Smoother = cms.string('GsfTrajectorySmoother_forPreId'),
+    TTRHBuilder = cms.string('WithAngleAndTemplate'),
+    ModelNames = cms.vstring([
+            'unbiased',
+            'ptbiased',
+            ]),
+    ModelWeights = cms.vstring([
+            'RecoEgamma/ElectronIdentification/data/LowPtElectrons/RunII_Fall17_LowPtElectrons_unbiased.xml.gz',
+            'RecoEgamma/ElectronIdentification/data/LowPtElectrons/RunII_Fall17_LowPtElectrons_displaced_pt_eta_biased.xml.gz',
+            ]),
+    ModelThresholds = thresholds("T"),
+    PassThrough = cms.bool(False),
+    UsePfTracks = cms.bool(True),
+    MinPtThreshold = cms.double(0.5),
```

Comment:
@bainbrid 
In the context of generic reco, where the intent is to use the tight working point, we can also consider to increase the MinPtThreshold.
What is the performance loss from going up to 0.7 and/or 0.9 GeV?

Reply:
Study done [here](https://github.com/cms-sw/cmssw/pull/25455#discussion_r246453718)